src/VisualStudio/Xaml/Impl/Features/DocumentSpan.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.Editor.Xaml.Features
+{
+    internal struct DocumentSpan
```

Comment:
When https://github.com/dotnet/roslyn/pull/12644/files goes in, you can use DocumentLocation as it is the exact same shape.